sdk/resourcemanager/api-specs.json

```diff
@@ -83,7 +83,7 @@
     "dir": "azure-resourcemanager-cosmos",
     "source": "specification/cosmos-db/resource-manager/readme.md",
     "package": "com.azure.resourcemanager.cosmos",
-    "args": "--tag=package-2023-04"
+    "args": "--tag=package-2023-03-15"
```

Comment:
wait a min, request is for package-preview-2023-03-15?

Reply:
https://github.com/Azure/sdk-release-request/issues/4188
Seems wrong..

Reply:
Yes, this is the request of [issue#4188](https://github.com/Azure/sdk-release-request/issues/4188).

Reply:
The issue asks
Readme Tag: package-preview-2023-03-15

Isn't it?

Reply:
The tag should be `package-preview-2023-03-15`, instead of `package-2023-03-15`? 
https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cosmos-db/resource-manager#tag-package-preview-2023-03-15

`package-2023-03-15` actually doesn't exist. I wonder why the code can be generated..

Reply:
here is no tag package-2023-03-15 in https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cosmos-db/resource-manager/readme.md

Reply:
Fixed to `"args": "--tag=package-preview-2023-03-15"` in the new version.